Sources/SWBUniversalPlatform/TestEntryPointGenerationTaskAction.swift

```diff
@@ -23,10 +23,37 @@ class TestEntryPointGenerationTaskAction: TaskAction {
     override func performTaskAction(_ task: any ExecutableTask, dynamicExecutionDelegate: any DynamicTaskExecutionDelegate, executionDelegate: any TaskExecutionDelegate, clientDelegate: any TaskExecutionClientDelegate, outputDelegate: any TaskOutputDelegate) async -> CommandResult {
         do {
             let options = try Options.parse(Array(task.commandLineAsStrings.dropFirst()))
-            try executionDelegate.fs.write(options.output, contents: #"""
+
+            var tests: [IndexStore.TestCaseClass] = []
+            var objects: [Path] = []
+            for linkerFilelist in options.linkerFilelist {
+                let filelistContents = String(String(decoding: try executionDelegate.fs.read(linkerFilelist), as: UTF8.self))
+                let entries = filelistContents.split(separator: "\n", omittingEmptySubsequences: true).map { Path($0) }.map {
+                    for indexUnitBasePath in options.indexUnitBasePath {
+                        if let remappedPath = generateIndexOutputPath(from: $0, basePath: indexUnitBasePath) {
+                            return remappedPath
+                        }
+                    }
+                    return $0
+                }
+                objects.append(contentsOf: entries)
+            }
+            let indexStoreAPI = try IndexStoreAPI(dylib: options.indexStoreLibraryPath)
+            for indexStore in options.indexStore {
+                let store = try IndexStore.open(store: indexStore, api: indexStoreAPI)
+                let testInfo = try store.listTests(in: objects)
+                tests.append(contentsOf: testInfo)
+            }
+
+            try executionDelegate.fs.write(options.output, contents: ByteString(encodingAsUTF8: """
             #if canImport(Testing)
             import Testing
             #endif
+            
+            \(testObservationFragment(testOutputPath: "TODO"))
```

Comment:
TODO still supposed to be here?

Reply:
Nope, fixed now so that this is provided via a CLI arg